packages/fsfoundation/src/ExtractPropertyNames.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Extracts the property names from a given object whose values are assignable to a given type
+ *
+ * Example:
```

Comment:
Shouldn't this be `@example` to follow the jsdocs spec?
I believe using `@example` would also provide the code snippet with syntax highlighting in tools like VS Code